dfdaemon/proxy/proxy.go

```diff
@@ -35,6 +35,18 @@ func WithCert(cert *tls.Certificate) Option {
 func WithHTTPSHosts(hosts ...*config.HijackHost) Option {
 	return func(p *Proxy) error {
 		p.httpsHosts = hosts
+		for _, host := range p.httpsHosts {
+			if host.ServerCertFile != "" && host.ServerKeyFile != "" {
+				logrus.Infof("load cert from local file, %s, %s", host.ServerCertFile, host.ServerKeyFile)
+				cert, err := tls.LoadX509KeyPair(host.ServerCertFile, host.ServerKeyFile)
+				if err != nil {
+					logrus.Errorf("failed to load self-signed certificate (%s, %s) for https hijacking, host: %s, err: %v",
```

Comment:
Do we need to return the err? Just comment to confirm.

Reply:
I think you are right, we should return err here